.github/workflows/aarch64-dit.yml

```diff
@@ -0,0 +1,37 @@
+name: aarch64-dit
+
+on:
+  pull_request:
+    paths:
+      - "aarch64-dit/**"
+      - "Cargo.*"
+  push:
+    branches: master
+
+permissions:
+  contents: read
+
+defaults:
+  run:
+    working-directory: aarch64-dit
+
+env:
+  CARGO_INCREMENTAL: 0
+  RUSTFLAGS: "-Dwarnings"
+
+jobs:
```

Comment:
FWIW I tried to see if it would work under `cross` in Linux, but unfortunately not:

```
---- tests::get stdout ----
thread 'tests::get' panicked at aarch64-dit/src/lib.rs:46:13:
DIT is not available on this CPU
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```